linter/src/main/java/linter/App.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package linter;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class App {
+    public static void main(String[] args) {
+        String filePath = "src/main/java/linter/resources/gates.js";
+        ArrayList<String> elist = findMissingSemiColons(filePath);
+        for(String error : elist){
+            System.out.println(error);
+        }
+    }
+    public static ArrayList<String> findMissingSemiColons(String path){
+        ArrayList<String> errorList = new ArrayList<>();
+        int lineNum = 0;
+
+        try
```

Comment:
This is closer to the C# way of doing things. In Java you typically put your opening `{` on the same line as the thing it's opening:

`try {`